lib/legato/management/account_summaries.rb

```diff
@@ -0,0 +1,42 @@
+module Legato
+  module Management
+    class AccountSummaries
+
+      extend Finder
+
+      def self.default_path
+        "/accountSummaries"
+      end
+
+      def path
+        "/accountSummaries"
+      end
+
+      attr_accessor :user, :account, :summary_web_properties, :summary_profiles
+
+      def initialize(attributes, user)
```

Comment:
@vedharish Do you mind taking a stab at this approach? Does it make sense?

What we get out of it then is the existing tree of Account => Property => Profile, using all of the existing classes. The summary is only used to get the data, and trigger the building of the tree.